src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -379,56 +379,18 @@ public static long getJSONLong(JSONObject object, String key) {
 
 	/************* LDAP lookup ****************/
 	private static final String OU_GROUPS = "ou=user groups,ou=groups,dc=umich,dc=edu";
-	private static final String ALLOW_USER_URLOVERRIDE = "allow.testUser.urlOverride";
+	private static final String ALLOW_USER_URL = "use.testUser.url";
```

Comment:
`use.testUser.url` this property was already existed in `application.properties` file on deluxe , but never put to use as we were just picking up the `test.RemoteUser` from the  properties file with out checking the test url flag disabled or enabled. Apart from that we had `allow.testUser.urlOverride' from code which had not connection/reference. So i have eliminated  one.